gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -197,217 +113,23 @@ def create_service_management_port(self, context, admin_context):
-        stitching_port = None
-        if stitching_subnet_id:
-            port_name = "stitching-port" + service_type
-            subnet = context.core_plugin.get_subnet(
-                                admin_context, stitching_subnet_id)
-            network_id = subnet['network_id']
-            stitching_port = self._create_port(
-                context, admin_context, admin_context.tenant_id,
-                port_name, network_id, subnet_id=stitching_subnet_id)
-
-            stitching_port_id = stitching_port['id']
-            stitching_port_ip = stitching_port[
-                                    'fixed_ips'][0]['ip_address']
-            return (stitching_port_id, stitching_port_ip)
-
-    def _add_extra_route(self, context, stitching_interface_ip,
-                         stitching_subnet_id, provider_subnet_id):
-        # TODO(Magesh): Pass subnet ID in filters
-        ports = context.core_plugin.get_ports(
-            context._plugin_context,
-            filters={'device_owner': ['network:router_interface'],
-                     'fixed_ips':  {'subnet_id': [stitching_subnet_id]}})
-        router_port = ports and ports[0] or {}
-        router_id = router_port.get('device_id')
-
-        if not router_id:
-            LOG.error(_("Router not attached to stitching network"))
-            return
-
-        subnet = context.core_plugin.get_subnet(
-            context._plugin_context, provider_subnet_id)
-        provider_subnet_cidr = subnet['cidr']
-        route_to_add = {"nexthop": stitching_interface_ip,
-                        "destination": provider_subnet_cidr}
-        self._add_router_route(context, router_id, route_to_add)
-
-    def _delete_extra_route(self, context, stitching_subnet_id,
-                           provider_subnet_id):
-        ports = context.core_plugin.get_ports(
-            context._plugin_context,
-            filters={'device_owner': ['network:router_interface'],
-                     'fixed_ips':  {'subnet_id': [stitching_subnet_id]}})
-        router_port = ports and ports[0] or {}
-        router_id = router_port.get('device_id')
-
-        if not router_id:
-            LOG.error(_("Router not attached to stitching network"))
-            return
-
-        provider_subnet = context.core_plugin.get_subnet(
-            context._plugin_context, provider_subnet_id)
-        provider_subnet_cidr = provider_subnet['cidr']
-        route_to_remove = {"destination": provider_subnet_cidr}
-        self._remove_router_route(context, router_id, route_to_remove)
-
     def _create_pt(self, context, ptg_id, name,
-                   port_id=None, admin_context=None):
-        if admin_context:
-            ctx = admin_context
-        else:
-            ctx = context._plugin_context
+                   port_id=None):
         pt = {'name': name,
               'description': '',
-              'tenant_id': ctx.tenant_id,
+              'tenant_id': context.plugin_context.tenant_id,
```

Comment:
For admin_owned chains that we are using, will this be the admin tenant_id or provider tenant_id?